test/pdfs/bug1425312.pdf.link

```diff
@@ -0,0 +1 @@
+https://www.unicode.org/charts/PDF/U11A50.pdf
```

Comment:
How about linking to the Internet Archive here, to avoid the file becoming unavailable, i.e. using the following link instead https://web.archive.org/web/20171116124714/https://www.unicode.org/charts/PDF/U11A50.pdf.